### PR TITLE
Prevent to close connection on dispatch

### PR DESCRIPTION
**Pull request by Serginyu**

### CODE REVIEW

**Comment by Serginyu** [comment]
Im trying to add the tests in the pipeline, but the tests are not working in local neither.

**Review by xaviapa — COMMENTED** [review]
What are the empty Makefile and docker-compose.yml files?

**Comment by Serginyu** [comment]
> What are the empty Makefile and docker-compose.yml files?

@xaviapa I was adding it to be able to have local environment and pipeline workflow. Now its finished!

File: src/Request.php [hunk 1]
```diff
@@ -62,16 +62,18 @@ public function connect()
...
-        $this->channel = $this->connection->channel();
     }
 
     /**
      * @throws Exception\Configuration
      */
     public function setup()
     {
-        $this->connect();
+        var_dump('test2');
```
Comment (xaviapa) [inline src/Request.php]: Forgotten var_dump!

Comment (bvis) [inline src/Request.php]: Are you sure you want to do this?

Comment (Serginyu) [reply src/Request.php]: :sweat_smile: removed, thanks!

**Review by xaviapa — APPROVED** [review]

**Review by joskfg — APPROVED** [review]

**Review by xaviapa — APPROVED** [review]

**Pull request merged**